common/djangoapps/student/tests/test_views.py

```diff
@@ -603,6 +605,105 @@ def test_email_settings_unfulfilled_entitlement(self, mock_email_feature, mock_c
         response = self.client.get(self.path)
         self.assertEqual(pq(response.content)(self.EMAIL_SETTINGS_ELEMENT_ID).length, 0)
 
+    @staticmethod
+    def get_html_for_view_course_button(course_key_string, course_run_string):
+        return '<a href="/courses/{}/info" class="enter-course " data-course-key="{}">View Course<span class="sr">&nbsp;{}</span></a>'.format(
```

Comment:
#NIT instead of passing in `course_key_string` twice, you can use keyed argument. Basically change this to 
`<a href="/courses/{course_key}/info" class="enter-course " data-course-key="{course_key}">View Course<span class="sr">&nbsp;{course_run}</span></a>`. Then you can use `.format` with keyed arguments

Reply:
Done